src/vtt-segment-loader.js

```diff
@@ -476,11 +476,12 @@ export default class VTTSegmentLoader extends SegmentLoader {
 
     const timestampmap = segmentInfo.timestampmap;
     const diff = (timestampmap.MPEGTS / ONE_SECOND_IN_TS) - timestampmap.LOCAL + mappingObj.mapping;
-
+    const mpegTsRollover = Math.floor((2**33) / ONE_SECOND_IN_TS);
```

Comment:
nit: to fix linter errors
```suggestion
    const mpegTsRollover = Math.floor((2 ** 33) / ONE_SECOND_IN_TS);
    
```